### PR TITLE
Update ReadMe for better font style

### PR DESCRIPTION
**Pull request by senbihan**
An updated font family to add in the settings.json file of VSCode for more accurate github stylish fonts for C/C++ or Python.

</img>
image

### CODE REVIEW

**Comment by thomaspink** [comment]
@senbihan Thank you for the pull
I agree on the `font-family`, but when i inspect the code on github the `font-size` is set to `12px`
screen shot 2018-01-31 at 09 52 52

Where did you see the `14px`?

**Comment by senbihan** [comment]
Oh. @thomaspink I am so sorry for that. I changed it for personal use, forgot to revert :P.
Sorry, man!

File: README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ This theme has been optimized for JavaScript, TypeScript, CSS, SASS(SCSS), HTML
 
 To get the full GitHub experience set the following settings in VSCode:     
 ```
-"editor.fontFamily": "'SF Mono', Menlo, Monaco, 'Courier New', monospace",
+"editor.fontFamily": "'SFMono-Regular','Consolas','Liberation Mono','Menlo','Courier','monospace'",
 "editor.lineHeight": 20,
-"editor.fontSize": 12
+"editor.fontSize": 14
```
Comment (thomaspink) [inline README.md]: Pls change back to 12 as discussed

Comment (senbihan) [reply README.md]: Done in my forked copy. Should I make a new PR?

**Comment by thomaspink** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (thomaspink) [reply README.md]: It's fine :)

**Pull request merged**